src/ucs/sys/string.h

```diff
@@ -363,11 +363,27 @@ static inline int ucs_string_is_empty(const char *str)
  * Allocates a path buffer of size PATH_MAX.
  *
  * @param buffer_p Pointer to the buffer.
+ *                 The buffer is allocated and should be released by the caller.
  * @param name     Name of the buffer for logging.
- * @return         UCS_OK on success, UCS_ERR_NO_MEMORY on failure.
+ *
+ * @return UCS_OK on success, UCS_ERR_NO_MEMORY on failure.
  */
 ucs_status_t ucs_string_alloc_path_buffer(char **buffer_p, const char *name);
 
+/**
+ * Allocate a buffer and format a path string into it.
```

Comment:
can we comment that the size of the buffer is always PATH_MAX?

Reply:
done